provider/source/data/debug/currency/extended@1/CAD/ar.json

```diff
@@ -1,5 +1,4 @@
 {
-  "display_names": {
-    "DisplayName": "دولار كندي"
-  }
+  "categorized_display_names": {},
```

Comment:
@robertbastian : if we already rejected the locales that does not have `other` pattern or displayname, why this case is exist ?

There is display_name but there is no `other` pattern

Reply:
We're rejecting ids that have neither a display name nor an other case.

Reply:
done